packages/fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -157,6 +166,7 @@ export class ForkChoice implements IForkChoice {
 
   getHead(): IBlockSummary {
     // balances is not changed but votes are changed
+    const timer = this.metrics?.forkChoiceFindHead.startTimer();
```

Comment:
I think you want to run this only inside the `if (!this.synced) {` block or we will time cache hits